src/docs/sdk/event-payloads/breadcrumbs.mdx

```diff
@@ -2,107 +2,149 @@
 title: Breadcrumbs Interface
 ---
 
-The Breadcrumbs Interface specifies a series of application events, or
-"breadcrumbs", that occurred before an event.
+Sentry uses _breadcrumbs_ to create a trail of events that happened prior to an issue. These events are very similar to traditional logs but can record more rich structured data.
```

Comment:
```suggestion
Sentry uses _breadcrumbs_ to create a trail of events that happened before an issue. These events are very similar to traditional logs but can record more rich structured data.
```

Reply:
I like your suggestion but I will leave it as it is to be consistent with the breadcrumbs description that we have on docs.sentry https://docs.sentry.io/enriching-error-data/breadcrumbs/

Reply:
+1 on adopting this intro. We should move to it for all breadcrumbs docs.